serviceAccount/charts/service-account/values.yaml

```diff
@@ -0,0 +1,15 @@
+# for discovering services and pods
+
+namespace: default
+frontend:
+  name: frontend
+  serviceAccountName: aws-main
+  repository: amazon/aws-cli
+
+backend:
+  name: backend
+  serviceAccountName: aws-main
+  repository: amazon/aws-cli
```

Comment:
seems like the both backend and frontend share the same serviceaccount. 
they should have different service account with different policies attached to these serviceaccounts